### PR TITLE
[browser] NativeAOT-LLVM support in browser-bench

### PR DESCRIPTION
**Pull request by maraf**
Build browser-bench for NativeAOT-LLVM with `-p:UsingNativeAOT=true`

### CODE REVIEW

**Review by pavelsavara — APPROVED** [review]

File: src/mono/sample/wasm/browser-bench/frame-main.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 "use strict";
 
-import { dotnet, exit } from './_framework/dotnet.js'
+import { dotnet, exit } from './dotnet.js'
```
Comment (radekdoulik) [inline src/mono/sample/wasm/browser-bench/frame-main.js:6]: I guess this will work because of the `WasmRuntimeAssetsLocation` change, right?

**Review by radekdoulik — APPROVED** [review]

File: src/mono/sample/wasm/browser-bench/frame-main.js (hunk 1 above)
Comment (maraf) [reply src/mono/sample/wasm/browser-bench/frame-main.js:6]: Exactly

File: src/mono/sample/wasm/browser-bench/Wasm.Browser.Bench.Sample.csproj [hunk 2]
```diff
@@ -8,8 +8,33 @@
...
     <RunAOTCompilation Condition="'$(EnableAOTAndTrimming)' != ''">$(EnableAOTAndTrimming)</RunAOTCompilation>
+    <WasmRuntimeAssetsLocation>./</WasmRuntimeAssetsLocation>
   </PropertyGroup>
 
+  <!-- NativeAOT -->
+  <PropertyGroup Condition="'$(UsingNativeAOT)' == 'true'">
+    <PublishTrimmed>true</PublishTrimmed>
+    <PublishDir>$(MSBuildThisFileDirectory)/bin/$(Configuration)/AppBundle</PublishDir>
+    <DebugType>none</DebugType>
+    <RestoreAdditionalProjectSources>$(RestoreAdditionalProjectSources);https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-experimental/nuget/v3/index.json</RestoreAdditionalProjectSources>
```
Comment (jkotas) [inline src/mono/sample/wasm/browser-bench/Wasm.Browser.Bench.Sample.csproj:19]: Can the reference to the `dotnet-experimental` feed be in the runtimelab branch only?

I understand that this is under a condition currently, but it makes auditing of dotnet/runtime more complicated and introduces risk of accidental dependencies of dotnet/rutime on runtimelab.

Comment (maraf) [reply src/mono/sample/wasm/browser-bench/Wasm.Browser.Bench.Sample.csproj:19]: It is used only in the "sample" project, but I can remove the reference from sources and @radekdoulik can configure it when running the bench on the infra

Comment (jkotas) [reply src/mono/sample/wasm/browser-bench/Wasm.Browser.Bench.Sample.csproj:19]: Yes, it would be nice. Thanks!

**Pull request merged**